bundles/org.openhab.core.config.discovery.addon/src/test/java/org/openhab/core/config/discovery/addon/tests/AddonSuggestionServiceTests.java

```diff
@@ -125,13 +126,13 @@ private void setupMockAddonInfoProvider() {
         AddonDiscoveryMethod hp = new AddonDiscoveryMethod().setServiceType(AddonFinderConstants.SERVICE_TYPE_MDNS)
                 .setMatchProperties(
                         List.of(new AddonMatchProperty("rp", ".*"), new AddonMatchProperty("ty", "hp (.*)")))
-                .setMdnsServiceType("_printer._tcp.local.");
+                .setParameters(List.of(new AddonParameter("mdnsServiceType", "_printer._tcp.local.")));
 
         AddonDiscoveryMethod hue1 = new AddonDiscoveryMethod().setServiceType(AddonFinderConstants.SERVICE_TYPE_UPNP)
                 .setMatchProperties(List.of(new AddonMatchProperty("modelName", "Philips hue bridge")));
 
         AddonDiscoveryMethod hue2 = new AddonDiscoveryMethod().setServiceType(AddonFinderConstants.SERVICE_TYPE_MDNS)
-                .setMdnsServiceType("_hue._tcp.local.");
+                .setParameters(List.of(new AddonParameter("mdnsServiceType", "_hue._tcp.local.")));
```

Comment:
ditto

Reply:
Left as is, as this would also require a local constant and the exact name of the parameter is not relevant for the test.